uwblueprint/supportive-housing
# Handling Record Edits on the Employee Directory Page

## Implementation description
* Deactivating / Activating employees is now shown immediately, not after refresh
* Adding employees instantly triggers toast, increments value and shows on modal, not after refresh
* Deleting employees instantly triggers toast, decrements value and shows on modal, not after refresh



## Steps to test
Deactivate / Activate:
1. If you have an active employee, click 'deactivate' and 'activate'. You should see button grey out OR turn back to green instantly, not after a refresh

Adding employee:
1. Toast message
2. Employee count increments by one instantly
3. Employee is seen on modal instantly
4. If employee count > numRecordsPerPage, the employee is added to the last page, but the user stays on the first page (As seen with record logs)

Delete Employee:
1. Toast message
2. Employee count decrements by one instantly
3. If you delete the last employee on the page, you are sent to the previous page



## What should reviewers focus on?
* 


## Checklist
